pkg/golinters/govet/govet.go

```diff
@@ -128,6 +128,7 @@ var (
 		sigchanyzer.Analyzer,
 		slog.Analyzer,
 		stdmethods.Analyzer,
+		stdversion.Analyzer,
```

Comment:
`stdversion` is missing in the list for go1.22.

It was added in go1.23:
https://github.com/golang/go/blob/go1.23.0/src/cmd/vet/main.go#L55-L87

Maybe we should remove `stdversion` from `defaultAnalyzer`, or change the comment to `https://github.com/golang/go/blob/go1.23.0/src/cmd/vet/main.go#L55-L87`?